[2024-06-06T04:09:28.713+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-06T04:09:28.752+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_testConnection_v1.test_s3_hook scheduled__2024-06-05T00:00:00+00:00 [queued]>
[2024-06-06T04:09:28.774+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_testConnection_v1.test_s3_hook scheduled__2024-06-05T00:00:00+00:00 [queued]>
[2024-06-06T04:09:28.781+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-06T04:09:28.847+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): test_s3_hook> on 2024-06-05 00:00:00+00:00
[2024-06-06T04:09:28.878+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=2373) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-06T04:09:28.882+0000] {standard_task_runner.py:63} INFO - Started process 2379 to run task
[2024-06-06T04:09:28.877+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_testConnection_v1', 'test_s3_hook', 'scheduled__2024-06-05T00:00:00+00:00', '--job-id', '309', '--raw', '--subdir', 'DAGS_FOLDER/TestConnectionDag.py', '--cfg-path', '/tmp/tmp0p4ndzit']
[2024-06-06T04:09:28.885+0000] {standard_task_runner.py:91} INFO - Job 309: Subtask test_s3_hook
[2024-06-06T04:09:29.135+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_testConnection_v1.test_s3_hook scheduled__2024-06-05T00:00:00+00:00 [running]> on host fe4af7f6193a
[2024-06-06T04:09:29.403+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_testConnection_v1' AIRFLOW_CTX_TASK_ID='test_s3_hook' AIRFLOW_CTX_EXECUTION_DATE='2024-06-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-05T00:00:00+00:00'
[2024-06-06T04:09:29.407+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-06T04:09:29.467+0000] {logging_mixin.py:188} INFO - <***.providers.amazon.aws.hooks.s3.S3Hook object at 0x7fa0a9b26c00>
[2024-06-06T04:09:29.507+0000] {base.py:84} INFO - Using connection ID 's3_conn' for task execution.
[2024-06-06T04:09:29.508+0000] {connection_wrapper.py:387} INFO - AWS Connection (conn_id='s3_conn', conn_type='aws') credentials retrieved from extra.
[2024-06-06T04:09:30.489+0000] {logging_mixin.py:188} INFO - ['pkm1.csv']
[2024-06-06T04:09:30.490+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-06T04:09:30.492+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-06T04:09:30.536+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_testConnection_v1, task_id=test_s3_hook, run_id=scheduled__2024-06-05T00:00:00+00:00, execution_date=20240605T000000, start_date=20240606T040928, end_date=20240606T040930
[2024-06-06T04:09:30.574+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-06T04:09:30.593+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-06T04:09:30.597+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-06T04:10:00.399+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-06T04:10:00.503+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_testConnection_v1.test_s3_hook scheduled__2024-06-05T00:00:00+00:00 [queued]>
[2024-06-06T04:10:00.575+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_testConnection_v1.test_s3_hook scheduled__2024-06-05T00:00:00+00:00 [queued]>
[2024-06-06T04:10:00.581+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-06T04:10:00.664+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): test_s3_hook> on 2024-06-05 00:00:00+00:00
[2024-06-06T04:10:00.681+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=2396) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-06T04:10:00.687+0000] {standard_task_runner.py:63} INFO - Started process 2404 to run task
[2024-06-06T04:10:00.684+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_testConnection_v1', 'test_s3_hook', 'scheduled__2024-06-05T00:00:00+00:00', '--job-id', '313', '--raw', '--subdir', 'DAGS_FOLDER/TestConnectionDag.py', '--cfg-path', '/tmp/tmpw3deacg4']
[2024-06-06T04:10:00.701+0000] {standard_task_runner.py:91} INFO - Job 313: Subtask test_s3_hook
[2024-06-06T04:10:00.877+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_testConnection_v1.test_s3_hook scheduled__2024-06-05T00:00:00+00:00 [running]> on host fe4af7f6193a
[2024-06-06T04:10:01.071+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_testConnection_v1' AIRFLOW_CTX_TASK_ID='test_s3_hook' AIRFLOW_CTX_EXECUTION_DATE='2024-06-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-05T00:00:00+00:00'
[2024-06-06T04:10:01.087+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-06T04:10:01.132+0000] {logging_mixin.py:188} INFO - <***.providers.amazon.aws.hooks.s3.S3Hook object at 0x7fa0a9c97ec0>
[2024-06-06T04:10:01.164+0000] {base.py:84} INFO - Using connection ID 's3_conn' for task execution.
[2024-06-06T04:10:01.168+0000] {connection_wrapper.py:387} INFO - AWS Connection (conn_id='s3_conn', conn_type='aws') credentials retrieved from extra.
[2024-06-06T04:10:01.988+0000] {logging_mixin.py:188} INFO - ['pkm1.csv']
[2024-06-06T04:10:01.989+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-06T04:10:01.991+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-06T04:10:02.035+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_testConnection_v1, task_id=test_s3_hook, run_id=scheduled__2024-06-05T00:00:00+00:00, execution_date=20240605T000000, start_date=20240606T041000, end_date=20240606T041002
[2024-06-06T04:10:02.066+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-06T04:10:02.091+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-06T04:10:02.093+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
